airbyte-webapp/src/hooks/services/useConnectionHook.tsx

```diff
@@ -239,9 +267,12 @@ export const useRemoveConnectionsFromList = (): ((connectionIds: string[]) => vo
 const useConnectionList = (payload: Pick<WebBackendConnectionListRequestBody, "destinationId" | "sourceId"> = {}) => {
   const workspace = useCurrentWorkspace();
   const service = useWebConnectionService();
+  const REFETCH_CONNECTION_LIST_INTERVAL = 60_000;
 
-  return useSuspenseQuery(connectionsKeys.lists(payload.destinationId ?? payload.sourceId), () =>
-    service.list({ ...payload, workspaceId: workspace.workspaceId })
+  return useSuspenseQuery(
+    connectionsKeys.lists(payload.destinationId ?? payload.sourceId),
+    () => service.list({ ...payload, workspaceId: workspace.workspaceId }),
+    { refetchInterval: REFETCH_CONNECTION_LIST_INTERVAL }
```

Comment:
I don't see any reason against re-fetching the connections every 60 seconds. That way we can show updated progress in the connection table. p99 latency for this endpoint is under 100ms, so I don't think it's a performance problem.